libraries/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -1,21 +1,62 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import * as path from 'path';
-
-import { FileSystem, JsonFile, type JsonObject, Path } from '@rushstack/node-core-library';
-
+import { JsonFile, type JsonObject, Path, type IPackageJson, FileSystem } from '@rushstack/node-core-library';
 import type { PackageManagerName } from './packageManager/PackageManager';
 import type { RushConfiguration } from './RushConfiguration';
-import { objectsAreDeepEqual } from '../utilities/objectUtilities';
+import { isMatch, merge, objectsAreDeepEqual } from '../utilities/objectUtilities';
```

Comment:
`import * as objectUtilities`

This way the function names will be like `objectUtilities.merge()` which is more readable and unambiguous for code searches.

Reply:
changed